deploy/charts/cert-manager/values.yaml

```diff
@@ -119,12 +120,12 @@ securityContext: {}
 
 # Container Security Context to be set on the controller component container
 # ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
-containerSecurityContext: {}
+containerSecurityContext:
```

Comment:
That's a good point. I hadn't paid attention to the original fix, but I don't think it's necessary to set a default containerSecurityContext. I'll revert that.

Reply:
I think maybe it can stay? I see that the an example in the official docs has both too https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod .
I was just curious how they interact.

Reply:
Yeah, but in those examples that show how to set certain securityContext settings at Pod level  (like runAsNonRoot) and then set container specific securityContext settings (like capabilities). I don't think it makes sense to set the same setting and value both in Pod and Container securityContext.